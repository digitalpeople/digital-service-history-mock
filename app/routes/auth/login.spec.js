import {
  postLogin,
} from './login';

jest.mock('./login-handler', () => ({
  postLoginHandler: 'postLoginHandlerMockFunction',
}));

describe('postLogin', () => {
  it('should call app.post with correct path', () => {
    expect.assertions(1);

    const app = {
      post: jest.fn(),
    };

    postLogin(app);

    expect(app.post.mock.calls[0]).toMatchSnapshot();
  });
});
