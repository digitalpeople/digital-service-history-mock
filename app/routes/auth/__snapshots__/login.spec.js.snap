// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`postLogin should call app.post with correct path 1`] = `
Array [
  "/auth/login/",
  "postLoginHandlerMockFunction",
]
`;
